fusilli-plugin/README.md

```diff
@@ -0,0 +1,26 @@
+# Fusilli Plugin
+
+Fusilli-Plugin: A Fusilli/IREE powered hipDNN plugin for graph JIT compilation.
+
+:construction: **This project is under active development, many things don't work yet** :construction:
+
+The plugin builds as a shared library (`fusilli_plugin.so`) providing a `hipDNN` [kernel engine plugin](https://github.com/ROCm/hipDNN/blob/develop/docs/PluginDevelopment.md#creating-a-kernel-engine-plugin) [API](https://github.com/ROCm/hipDNN/blob/839cf6c4bc6fe403d0ef72cb5d7df004e2004743/sdk/include/hipdnn_sdk/plugin/EnginePluginApi.h).
+
+## Developer Guide
+
+### Setup
+
+For the time being, `fusilli-plugin` setup relies on / builds on [Fusilli setup](../sharkfuser/#setup).
+Keeping the projects in sync prevents "works on my machine" style bugs.
+Requirements that are unique to `fusilli-plugin`, `hipDNN` and `googletest` for
+example, are fetched configured and built as part of `fusilli-plugin` build.
+
+After following steps in [Fusilli Setup](../sharkfuser/#setup), build and test
```

Comment:
Curious, where does `../sharkfuser/#setup` point to? Don't you need the `README.md` in there somewhere?

Reply:
Hmm yeah, it seemed like this worked in vscode but it doesn't look like it does.